apps/web/src/app/app.component.ts

```diff
@@ -60,6 +60,8 @@ export class AppComponent implements OnDestroy, OnInit {
   private isIdle = false;
   private destroy$ = new Subject<void>();
 
+  loading = false;
```

Comment:
suggestion (non-blocking): 
```suggestion
  protected loading = false;
```